data/changelog.txt

```diff
@@ -265,84 +166,19 @@ Vehicles: make shock absorbers actually absorb shock damage.
-Fix droping NO_DROP items from destroyed container
-Fixed pushing past enemies on stairs in a mech would render the mech inoperable
-Fix incorrectly resuming interrupted vehicle activity when a different vehicle activity is initiated
-Fix turret missing eyes
-Allow canceling spilling when eating from sealed container
-Fixed loading deployable turrets with ammo
-Disallowed the disassembly of gas mask filters
-Air is invisible
 Improve invalidation of max_populated_zlev cache, fixes lighting in some teleportation scenarios.
-fixed a misplaced tile in house_19
-Fix crafting with wielded item and force unarmed style
 Prevent dodging or blocking while insensible.
-Brass catcher catches casings
-Remove extra wristwatch from Bionic assassin
-Fix wizard tower orientation.
-Fix basement nest orientation from #39845.
-prevents map extras from spawning in church steeple.
-Several transparency related issues
-Misc spilling and inventory fixes
-Add Character sight tests, fix Ursine eyes
-Faulty engine could be started by changing fuel selection.
-fix map radiation deserialization
-fix warnings in the log for when monster is on the border of the map and its goal is outside
-Use game start for survival time on tombstone
-Unseal and spill containers in dropping/stashing activity
-Fix encumbrance JSON for jumpsuits, stop travis erroring
-mansion wrong wall terrain
-MBR Vests not being migrated in Fujis Military Professions
-Fix crash and result when grinding chitin in a mill
-Fix speedloader code and related json
-Unseals containers of components used in crafting
-Changes frying to be closer to real world frying substrate absorption
-Added longest_side (82cm) to chainsaw (off) JSON
-Fixes stone adze being able to saw tree trunk into planks through the construction menu.
-Inserting an item stack into a container tries to fill it up instead of put in all charges
-Added installation time to gunmods
-Minor translation fixes
-Inserting full item stacks doesn't leave behind empty stacks
-Check replace add_msg by add_msg_if_player in suffer.cpp
-Fix chance_of_success for saving cyborgs
-Fix empty vehicle symbol causing displaced console output
-Fix apparent light symmetry when sun shines through a single-tile hole in the roof
+Fix apparent light symmetry when sun shines through a single-tile hole in the roof.
 
 ## Performance:
 Add option to enable batching when it is available in SDL.
 Skip unnecessary weather calculations for root cellar.
 Cache drop_invalid_inventory and weight_carried.
-Check for safemode emptiness before calling monster check.
-Code optimizations reported by static code analysis
-Don't serialize default pocket settings
-Cache clairvoyance field
-Remove  second (excessive) call to `update_visibility_cache`
-improved performance in several places, such as sunlight casting
-Performance improvements related to map::process_fields
-Increase performance when character is suffering in sunlight
-Improve cache invalidation logic for Field of View.
```

Comment:
This is one of those ones that can be important later on to emphasize (for easier finding).